### DISCUSSION TITLE
I need help with the in-range: and out-of-range: modifiers in inputs

### DISCUSSION THREAD

**Discussion by DoudGaya** [tailwindlabs/tailwindcss#9466] [Help]
When I add autility styling to the ( **in-range:** ) or the (**:out-of-range**) modifiers for inputs it doesn't seem to be working. So I tried using the appearance-none utility, which removes the default styling but how can I then add my custom range and out-of-range because every styling I add to the modifier doesn't seems to work.

**Comment by wongjn** [comment]
Have you added `min` or `max` attributes to the `<input>` element(s)? If so and it still doesn't work, perhaps if you could send through a [Tailwind Play](https://play.tailwindcss.com/) reproducing your issue, we can help troubleshoot it further for you.

**Reply by DoudGaya** [reply]
https://play.tailwindcss.com/5IS21DE1ue. This is what I have with the default styling. 
https://play.tailwindcss.com/EO9jdai0bA and if I add the `appearance-none` class I can only see the `in-range:` styling added and I want both grey and white background like the one on the default styling

**Reply by wongjn** [reply]
`range` elements can never be selected with the `:out-of-range` pseudo-class, since the `min` and `max` values correspond to the minimum and maximum values selectable with the input element and thus one can never select a value outside this. With regards to the background and border, you should be able to use the standard `background-color` and `border` CSS properties to style the main track of the range slider, though you should check how it looks on other browser rendering engines, as form element appearance can vary between the different rendering engines.
